resources/function_help/json/split_geometry

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "split_geometry",
+  "type": "function",
+  "groups": ["GeometryGroup"],
+  "description": "Returns a valid geometry or an empty geometry if the geometry could not be made valid.",
+  "arguments": [{
+    "arg": "baseGeometry",
+    "description": "The geomety(ies) to split"
+  },
+  {
+    "arg": "splittingGeometry",
```

Comment:
what about `split_geometry(geometry, split_by)`? shorter, but I think, still meaningful.

Reply:
seems reasonable, though I'm worried some may provide a layer name instead of a geometry. But the error handler should pop up for that.